Docker.DotNet/project.json

```diff
@@ -12,6 +12,7 @@
         "portable-dnxcore50+net45+win8+wp8+wpa81"
       ],
       "dependencies": {
+        "System.Buffers": "4.0.0-rc3-23829",
```

Comment:
We actually have the .NET Core 1.0 release as of a few days ago. We should likely update to that but I can do it as part of another PR if you don't want to change them now.